naga/tests/out/wgsl/phony_assignment.wgsl

```diff
@@ -0,0 +1,7 @@
+@group(0) @binding(0) 
+var<uniform> binding: f32;
+
+@compute @workgroup_size(1, 1, 1) 
+fn main(@builtin(global_invocation_id) id: vec3<u32>) {
+    _ = binding;
```

Comment:
Whoa, how come `_ = a;` does away, but `_ = binding;` does not?

Reply:
because `let x = a` would also get optimized away (by compat), because it's constant (and has no side effects so this is actually ok).